SU-SWS/stanford_profile_helper
# D8CORE-2497: setting max-width to 980 like the cards

# READY FOR REVIEW

# Summary
- Setting a fixed max-width to single CTA Image cards.

# Needed By (Date)
- EOD Wed

# Urgency
- High

# Steps to Test

1. Pull in branch and build
2. Add a single CTA Image to a row.
3. Verify that the max-width is 980px on full width and not pages.

# Affected Projects or Products
- SOE Theming

# Associated Issues and/or People
- D8CORE-2497

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
